topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -0,0 +1,290 @@
+>
+> Thanks a lot in advance for your feedback!
+>
+{title="Early Access Status" style="warning"}
+
+## Requirements
+
+The following platforms and environments are supported:
+
+- IntelliJ Platform: 2022.3 and later
+- Java Runtime: 17 and later
+- Gradle: 8.1 and later
+
+## Usage
+
+To use the current Early Access Preview snapshot versions, add to your <path>settings.gradle.kts</path> file:
+
+```kotlin
+pluginManagement {
+  repositories {
+    maven("https://oss.sonatype.org/content/repositories/snapshots/")
+    gradlePluginPortal()
+  }
+}
+```
+
+In the <path>build.gradle.kts</path> file, replace the existing reference to the [](tools_gradle_intellij_plugin.md) plugin (`org.jetbrains.intellij`) with:
+
+```kotlin
+plugins {
+  id("org.jetbrains.intellij.platform") version "%intellij-platform-gradle-plugin-version%"
+}
+```
+
+### Available Subplugins
+
+The plugin has a new ID: `org.jetbrains.intellij.platform`, but also was split into subplugins.
+This approach allows you to choose between applying all configurations and tasks at once, or applying them separately, e.g., to only use an IntelliJ Platform SDK dependency without creating any tasks.
+
+#### org.jetbrains.intellij.platform
+
+{id="plugin.platform"}
+
+This plugin applies all project-level plugins, which brings the fully-flagged tooling for plugin development for IntelliJ-based IDEs.
+
+It includes [](#plugin.core) and [](#plugin.tasks) subplugins.
+
+#### org.jetbrains.intellij.platform.core
+
+{id="plugin.core"}
+
+Base plugin registers all necessary custom configurations and dependencies transformers used for handling IntelliJ Platform SDK, JetBrains Runtime, CLI tools, or other plugins when used as dependencies.
+
+#### org.jetbrains.intellij.platform.tasks
+
+{id="plugin.tasks"}
+
+Tasks plugin registers and preconfigures all tasks introduced by the IntelliJ Platform Gradle Plugin.
+It can be omitted when referring to any IntelliJ Platform SDK dependencies without invoking tasks on project submodules.
+
+#### org.jetbrains.intellij.platform.settings
+
+{id="plugin.settings"}
+
+Settings plugin is required in <path>settings.gradle.kts</path> when using `RepositoryHandler` extensions when declaring the IntelliJ Maven repository (see [](intellij_artifacts.md)) for the `dependencyResolutionManagement` Gradle mechanism.
+
+## Configuration
+
+> Auto-completion, Quick Documentation, and other code insight features are available for many extension functions and properties.
+>
+{title="Exploring Configuration Options"}
+
+### Setting up repositories
+
+{id="configuration.repositories"}
+
+All IntelliJ Platform SDK artifacts are available via IntelliJ Maven repositories (see [](intellij_artifacts.md)), which exist in three variants:
+
+- releases
+- snapshots
+- nightly (only select artifacts)
+
+All required repositories for the project must be declared explicitly within the `intellijPlatform` extension:
```

Comment:
Maybe a better example would be: Assume you want to build your plugin against a release version of the IntelliJ Platform and you also have a dependency on a plugin from the Marketplace, then you would declare the following repositories: